src/commands/server/restore.ts

```diff
@@ -85,10 +115,208 @@ export default class Restore extends Command {
   private getRestoreTasks(flags: any): Listr.ListrTask[] {
     return [
       {
-        title: 'Scheduling restore...',
+        title: 'Detecting existing operator version...',
+        enabled: () => flags.version || flags.rollback || flags['backup-cr'],
+        task: async (ctx: any, task: any) => {
+          const kube = new KubeHelper(flags)
+          let operatorDeploymentYaml = await kube.getDeployment(OPERATOR_DEPLOYMENT_NAME, flags.chenamespace)
+          if (!operatorDeploymentYaml) {
+            // There is no operator deployment in Che namespace
+            // Check if the operator is in all namespaces mode
+            operatorDeploymentYaml = await kube.getDeployment(OPERATOR_DEPLOYMENT_NAME, DEFAULT_OPENSHIFT_OPERATORS_NS_NAME)
+            if (!operatorDeploymentYaml) {
+              // Still no operator deployment found
+              ctx.currentOperatorVersion = ''
+              task.title = `${task.title} operator is not deployed`
```

Comment:
Throw an error?

Reply:
No, it is a case. We can and should proceed.